frontend/src/components/RecipeDraft.js

```diff
@@ -23,18 +59,38 @@ export default function RecipeDraft({recipe, onRecipeNameUpdated, onIngredientsU
                     onChange={onRecipeNameUpdated}
                 />
                 {ingredients.map(ingredient => (
-                    <IngredientEditor key={ingredient.id} ingredient={ingredient} onChange={onIngredientUpdated}/>
+                    <IngredientEditor
+                        key={ingredient.id}
+                        ingredient={ingredient}
+                        onChange={onIngredientUpdated}
+                        onDelete={onDeleteIngredient}
+                    />
                 ))}
-                <Button type="submit" >Speichern</Button>
+                <ButtonWrapper>
+                    <IconButton type="button" onClick={onAddIngredient}>
+                        <AddCircleOutlineOutlinedIcon/>
+                    </IconButton>
+                    <Button type="submit"><CheckOutlined/></Button>
+                </ButtonWrapper>
             </form>
         </>
     )
 }
 
+function uuid() {
```

Comment:
maybe it would be easier to just use the builtin uuid functionality

Reply:
easier than copy&paste? :)